packages/builder/src/stores/builder/tables.ts

```diff
@@ -99,28 +133,33 @@ export function createTablesStore() {
     const tableIdsToFetch = [...new Set([...newTableIds, ...oldTableIds])]
     // too many tables to fetch, just get all
     if (tableIdsToFetch.length > 3) {
-      await fetch()
+      await this.fetch()
     } else {
-      await Promise.all(tableIdsToFetch.map(id => singleFetch(id)))
+      await Promise.all(tableIdsToFetch.map(id => this.singleFetch(id)))
     }
     return savedTable
   }
 
-  const deleteTable = async table => {
+  async delete(table: { _id: string; _rev: any }) {
```

Comment:
Why does `_rev` need to be any here - is `_rev?: string` not sufficient?

Reply:
Yup, mistake. Done. It should always be required, and it looks like there is a single usage were we are passing the full table